.devcontainer/devcontainer.json

```diff
@@ -73,7 +50,10 @@
 				"yzhang.markdown-all-in-one",
 				"visualstudioexptteam.vscodeintellicode",
 				"ymotongpoo.licenser",
-				"editorconfig.editorconfig"
+				"editorconfig.editorconfig",
```

Comment:
Are we still using the editor config? Or can we move that out? I.e. also the .editorconfig in the root directory. As far as I understand this is only relevant if we are using multiple editors which we are not.

Reply:
So for end of line settings, you can add this to vscode settings:
```
    "files.eol": "\n"
```
When it comes to the encoding, the default is already utf-8 in vscode. So no need to set that.

So I guess we can remove the editorconfig

Reply:
@KerstenBreuer please see [0854d59](https://github.com/ghga-de/microservice-repository-template/pull/171/commits/0854d59fb63291ca385c024c590bf55acf584cf6)

Reply:
Perfect, thanks